### ISSUE TITLE
link refprop version 10 in windows 10

### ISSUE THREAD

**Issue by narcismech68** [usnistgov/REFPROP-issues#112]
If you have an issue to report, please continue and fill out the applicable sections below.  The details provided will help to resolve this issue as quickly as possible.

### Prerequisites

**NOTE**: Be aware that all issues are publicly accessible and viewable.  Thus please do not post any code or other content that is protected intellectual property or under copyright.

Before posting an issue, please:  
    * Check the [FAQ](https://pages.nist.gov/REFPROP-docs/) for common solutions.  
    * Check that your issue isn't already filed under [REFPROP Issues](https://github.com/usnistgov/REFPROP-issues/).  
    * Check the [documentation](http://refprop-docs.readthedocs.io/en/latest/).  

You can also use markdown to format your issue: [GitHub guide on Markdown](https://guides.github.com/features/mastering-markdown/). If you have code snippets, please use a code block to ensure that the formatting remains legible in the web interface.  For instance, surround your code in triple backticks:

``` python
print('Hello world!')
```

### Description

[Description of the issue]

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

**Expected behavior:** [What you expect to happen]

**Actual behavior:** [What actually happens]

### Versions

**REFPROP Version:** [REFPROP version you are using]  
**Operating System and Version:** [OS you are running REFPROP on]  
**Access Method:** [How you are accessing REFPROP (i.e. Python, other wrapper)]  

### Additional Information

If possible, please post examples and/or screenshots of the issue.

**Comment by narcismech68** [comment]
Halo!

I try to link the refprop 10 to matlab in a windows 10 Laptop.
I have tried to do so based on this website, step by step:

https://github.com/usnistgov/REFPROP-wrappers/blob/master/wrappers/MATLAB/README.md

but I couldn't run these commands:
----------------------------------------------------------------------------------
[v,e] = pyversion; system([e,' -m pip install --user -U ctREFPROP'])

[v,e] = pyversion; system([e,' -m easy_install pip'])

RP = py.ctREFPROP.ctREFPROP.REFPROPFunctionLibrary('C:\Program Files (x86)\REFPROP')
------------------------------------------------------------------------------------

I installed ctRefprop by using this:
pip install ctREFPROP
it seems that I install the package, but above-mentioned commands are not working in any environment which I tested. I don't know which environment is the proper one to use those commands. 

I really appreciate it if you give me a hint, so I be able to use refprop in my new windows...

Deep regards,
Narges

**Comment by ianhbell** [comment]
You should copy-paste the following commands into your MATLAB shell (please show the same commands from your installation):
```  matlab
>> [v,e] = pyversion; system([e,' -m pip install --user -U ctREFPROP'])
Cache entry deserialization failed, entry ignored 
Requirement already up-to-date: ctREFPROP in d:\anaconda\envs\py36\lib\site-packages (0.7.2) 
wxpython 4.0.2 requires PyPubSub, which is not installed. 
You are using pip version 10.0.1, however version 19.0.2 is available. 
You should consider upgrading via the 'python -m pip install --upgrade pip' command. 

ans =

     0

>> RP = py.ctREFPROP.ctREFPROP.REFPROPFunctionLibrary('C:\Program Files (x86)\REFPROP')

RP = 

  Python REFPROPInstance with properties:

    managed_handle: [1×1 py.NoneType]
               dll: [1×1 py.ctypes.WinDLL]

    <ctREFPROP.ctREFPROP.REFPROPInstance object at 0x00000000F8A6D9B0>

```

**Comment by narcismech68** [comment]
Thanks Ian!

It was obvious and I did it.
because of that I cleared the massage I sent to the forum last night.
However it seems there is a problem with my refprop.
it is a dll beta version i get it from Eric Lemmon.
I had an error which says: the refprop dll is expired...
and I don't know what to do.

I want to buy it but I should wait until I be a Phd student in another
country.
Then I would have financial support to buy.

Deep Regards
Narges
